karton/core/config.py

```diff
@@ -44,6 +44,20 @@ def __init__(self, path: Optional[str] = None) -> None:
         if not self.config.has_section("redis"):
             raise RuntimeError("Missing Redis configuration")
 
+    @staticmethod
+    def from_dict(config_dict):
+        # Create the object. This will behave in a normal way, by scaning
+        # the filesystem. If it fails, RuntimeError will be raised.
+        # We can ignore it. Created ConfigParser will be overwritten by ours
+        try:
+            config = Config()
+        except RuntimeError:
+            pass
```

Comment:
What's the point of this :thinking:.

Config won't even be defined if RuntimeError is raised. Try:

```
>>> try:
...     kot = 1/0
... except:
...     pass
...
>>> print(kot)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'kot' is not defined
```

Unless I'm wrong and somehow half-initialised objects can exist in Python?